src/utils.hpp

```diff
@@ -5,7 +5,9 @@
 #include <string_view>
 
 namespace Exiv2 {
```

Comment:
Curious. Is this public API?

Reply:
Good point. We should hide those symbols in the `Exiv2::Internal` namespace. It is not part of the Public API